amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/AsyncPlugin.scala

```diff
@@ -35,10 +35,7 @@ sealed trait AsyncPlugin extends OasLikePlugin {
               wrapped: ParserContext,
               ds: Option[AsyncWebApiDeclarations] = None): AsyncWebApiContext
 
-  override def parse(document: Root,
-                     parentContext: ParserContext,
-                     platform: Platform,
```

Comment:
I saw Platform being used as a parameter in several places, why do we have this if Platform is a Singleton for the distribution? Is this usage legacy?

Reply:
Introduced this changed as it was a parameter that was unused in all plugin implementations. As you mention, this most likely was a legacy usage.